cli/combined-shim.yml

```diff
@@ -16,6 +16,11 @@ builds:
       - arm64
     goamd64:
       - v1
+    ignore:
```

Comment:
I think we still want to build the windows/arm package, it just has x86 binaries in it.